android-app/app/src/test/java/de/dhbw/ka/se/fibo/HelpersUnitTest.java

```diff
@@ -4,15 +4,21 @@
 
 import org.junit.Test;
 
+import java.math.BigDecimal;
+
 /**
  * Example local unit test, which will execute on the development machine (host).
  *
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
 public class ExampleUnitTest {
```

Comment:
```suggestion
public class HelpersUnitTest {
```

This needs to be changed so that it fits with the changed filename. It's necessary that atleast one class exists in a java file with the same name like the filename